Code/Editor/Core/QtEditorApplication.cpp

```diff
@@ -426,8 +429,16 @@ namespace Editor
 
     void EditorQtApplication::InstallEditorTranslators()
     {
-        m_editorTranslator =        CreateAndInitializeTranslator("editor_en-us.qm", ":/Translations");
-        m_assetBrowserTranslator =  CreateAndInitializeTranslator("assetbrowser_en-us.qm", ":/Translations");
+        QTextCodec::setCodecForLocale(QTextCodec::codecForName("utf-8"));
+
+        QStringList translatorNames = {
+            "Editor"
+        };
+
+        for (QString trans : translatorNames)
+        {
+            m_translators.append(AzToolsFramework::CreateAndLoadTranslator(trans));
+        }
```

Comment:
In theory we can initialize the translators via the Module class hook within an Internationalization Gem.

The instantiated CppToolGem template [*EditorModule.cpp](https://github.com/o3de/o3de/blob/development/Templates/CppToolGem/Template/Code/Source/Tools/%24%7BName%7DEditorModule.cpp#L30-L41) would be a good place to hook in the Qt Translators